scripts/ci

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
 set -e
 
+if [ ! -z ${1} ]; then
```

Comment:
It is added some that `./scripts/ci` can take arguments and only run for specific chart